src/main/data/studios.json

```diff
@@ -971,4 +973,4 @@
     "code": "ZM",
     "description": "ZOOM-ENTERTAINMENT"
   }
-]
+]
```

Comment:
Thoughts on whether the canonicalized form should have a final newline? My understanding is that [the POSIX definition of a line](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206) requires it.